core/src/main/java/nu/yona/server/analysis/entities/IntervalActivity.java

```diff
@@ -0,0 +1,53 @@
+package nu.yona.server.analysis.entities;
+
+import java.time.ZonedDateTime;
+import java.util.Date;
+import java.util.UUID;
+
+import javax.persistence.Entity;
+
+import nu.yona.server.entities.EntityWithID;
+import nu.yona.server.goals.entities.Goal;
+
+@Entity
+public abstract class IntervalActivity extends EntityWithID
+{
+	private UUID goalID;
+	private UUID userAnonymizedID;
+	private ZonedDateTime zonedStartTime;
```

Comment:
Just name it ``startTime``. Some day, we need to switch all date/times to "zoned".